system/lib/compiler-rt/lib/asan/asan_emscripten.cpp

```diff
@@ -74,7 +76,7 @@ INTERCEPTOR(int, pthread_create, void *thread,
     StopInitOrderChecking();
   GET_STACK_TRACE_THREAD;
   int detached = 0;
-  if (attr)
+  if (attr && attr != __ATTRP_C11_THREAD)
```

Comment:
are the changes in this file updates from newer compiler-rt?

Reply:
No, this is a change to make compiler-rt compatible with musl's C11 thread implementation.   I'll add to the readme

Reply:
Actually this file doesn't exist upstream..  so I don't think we need to modify the readme